pyfar/classes/audio.py

```diff
@@ -1581,16 +1581,13 @@ def _get_arithmetic_data(data, domain, cshape, matmul, audio_type,
     if isinstance(data, (Signal, TimeData, FrequencyData)):
         data_ = data.copy()
```

Comment:
I think we could copy in place, i.e., `data = data.copy()` to avoid doubling the data.